scripts/package

```diff
@@ -15,12 +15,18 @@ cp ../bin/pipeline ./pipeline/
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
+#Tag for dependent images
+PIPELINE_UI_TAG=v0.1.1
+JENKINS_BOOT_TAG=v0.1.1
```

Comment:
https://github.com/biblesyme/jenkins-boot/releases
here is the release for jenkins-boot